packages/browsers/src/install.ts

```diff
@@ -98,6 +98,17 @@ export interface InstallOptions {
    * @defaultValue `false`
    */
   forceFallbackForTesting?: boolean;
+
+  /**
+   * Whether to attempt to install system-level dependencies required
+   * for the browser.
+   *
+   * Currently, only supported for Debian and Ubuntu and only for Chrome.
```

Comment:
I think we should try to "say" the Browser first as that is more important than the Platform.
```suggestion
   * Only supported for Chrome on Debian or Ubuntu.
```